worker/index.test.js

```diff
@@ -0,0 +1,25 @@
+const { unstable_dev } = require("wrangler");
```

Comment:
Please file a follow-up issue to have this get implemented properly.  We might decide to remove this from your PR for now, and do it there.